services/wallet/controllers_v3.go

```diff
@@ -713,3 +816,10 @@ func DisconnectCustodianLinkV3(s *Service) func(w http.ResponseWriter, r *http.R
 		return handlers.RenderContent(ctx, map[string]interface{}{}, w, http.StatusOK)
 	}
 }
+
+func strOr(a string, b string) string {
+	if a == "" {
+		return b
+	}
+	return a
+}
```

Comment:
Not sure if we need this. It's used only in one place and distracts from the existing convention:

```golang
a := thing
if a == "" {
	a = "default"
}
```